docs/data/material/components/material-icons/material-icons.md

```diff
@@ -36,7 +36,31 @@ pnpm add @mui/icons-material @mui/material @emotion/styled @emotion/react
 
 See the [Installation](/material-ui/getting-started/installation/) page for additional docs about how to make sure everything is set up correctly.
 
-<hr/>
+## Basics
+
+Material Icons use the Material UI [SVG Icon](/material-ui/icons/#svgicon/) component under the hood, so they render without any customization, and feature several props for quickly customizing styles.
+
+{{"demo": "BasicMaterialIcon.js"}}
+
+:::info
+Visit [the SVG Icon component section in the Icon doc](/material-ui/icons/#svgicon/) for further information about its other props, as well as how to use it with other icon libraries.
+:::
+
+### Size
+
+Use the `fontSize` prop to toggle between small, medium (default, 24x24px), or large icon sizes.
+You can also use the `sx` prop to pick arbitrary values that are outside of this built-in scale.
+
+{{"demo": "SizeMaterialIcon.js"}}
+
+### Color
+
+Use the `color` prop to choose a palette key from the theme.
+It defaults to the `main` value of its respective palette.
+
+{{"demo": "ColorMaterialIcon.js"}}
```

Comment:
I wouldn't document this, It feels like it doesn't teach people that it's https://deploy-preview-40445--material-ui.netlify.app/material-ui/icons/#svgicon